doc/lua_api.txt

```diff
@@ -5671,6 +5678,20 @@ This is basically a reference to a C++ `ServerActiveObject`
 * `hud_set_hotbar_selected_image(texturename)`
     * sets image for selected item of hotbar
 * `hud_get_hotbar_selected_image`: returns texturename
+* `set_minimap_modes({modes}, selected_mode)`: Sets available minimap modes and
+  selected mode number (existing modes are replaced). Modes is a table of mode
+	descriptions. Mode description is a table with four fields :
+    * `type`: Available type:
+      * `off`: Minimap off
+      * `surface`: Minimap in surface mode
+      * `radar`: Minimap in radar mode
+      * `texture`: Minimap displays texture given in `texture` field instead of
+        terrain map
+    * `label`: Label associated with this mode (displayed on F9)
```

Comment:
F9 for minimap isn't a fixed keybind, and so using the name of the key isn't a good idea. Would it be better if the name of the action is used instead? e.g. `(displayed when pressing "Minimap" key)`

Reply:
You're right ! Fixed :)

Reply:
Thanks!